src/lua/init.h

```diff
@@ -43,6 +43,14 @@ extern bool start_loop;
 
 extern struct lua_State *tarantool_L;
 
+/**
+ * Sets an internal variable to true when Tarantool goes background.
+ *
+ * @return  None
```

Comment:
I think
```
/** Sets an internal variable to true when Tarantool goes background. */
```
should be enough since Doxigen is not required for function description. Especially for such simple functions.